src/pkcs15init/profile.c

```diff
@@ -1985,24 +1985,40 @@ static struct block	root_ops = {
    "root",		process_block,	NULL,		root_blocks
 };
 
+static void
+get_inner_word(char *str, char word[64]) {
```

Comment:
I'm not sure whether the macros can be nested inside another expression; the problematic one found by OSS-Fuzz looks like `odf-size=($odf-size�v$` (with some non-alphanumeric byte).

Reply:
I think this is mostly valid use case for example in the myeid profile object, there are macros used as part of the strings (indeed unless it is recursive).

Reply:
Ok, thanks for checking.